doc/conf.py

```diff
@@ -141,6 +141,10 @@ def _check_dependencies():
     'within_subsection_order': gallery_order.subsectionorder,
     'remove_config_comments': True,
     'min_reported_time': 1,
+    # Default thumbnail size (400, 280)
+    # Default CSS rescales (160, 112)
+    # Size is decreased to reduce webpage loading time
+    'thumbnail_size': (280, 196),
```

Comment:
Why a factor of 1.75 instead of 2?   Not sure of the practical benefit of this since our hosting provider compresses the pngs for us, but it looks fine even at 1.75...

Reply:
Visually, it would be too much.

Reply:
Not sure I follow...  There are *lots* of high-DPI/retina displays in the wild, so CSS 160 is really 320 pixels and hence a factor of 2 makes sense.